src/aatoolbox/config/countryconfig.py

```diff
@@ -31,11 +31,45 @@ class CodABConfig(BaseModel):
     custom_layer_names: Optional[list]
 
 
+class FewsNetConfig(BaseModel):
+    """FEWS NET configuration.
+
+    Parameters
+    ----------
+    region_name: str
+        Name of the region the country belongs to. Needed to download the
+        regional FEWS NET data
+    """
+
+    region_name: str
+
+    @validator("region_name")
+    def regionname_valid(cls, v):
+        """Check that regionname is one of the valid ones."""
+        valid_regionnames = [
```

Comment:
@turnerm hmm I am wondering how we can document clearly what the valid regionnames are.